code/modules/surgery/revival.dm

```diff
@@ -29,7 +29,7 @@
 	name = "shock body"
 	implements = list(/obj/item/shockpaddles = 100, /obj/item/melee/baton = 75, /obj/item/gun/energy = 60)
 	repeatable = TRUE
-	time = 120
+	time = 50
```

Comment:
```suggestion
	time = 5 SECONDS
```

Reply:
There's quite a few other surgeries still using deciseconds. Should they be cleaned up as well?

Reply:
Yes